spec/features/edit_a_pom_spec.rb

```diff
@@ -44,22 +45,36 @@
     expect(page).to have_content('Select number of days worked')
   end
 
-  it "makes an inactive POM active", vcr: { cassette_name: 'prison_api/edit_poms_activate_pom_feature' } do
-    # This doesn't do what it appears to - the URL is wrong so we're not editing an inactive POM... :-(
-    visit "/prisons/LEI/poms#inactive"
-    click_link 'Moic Integration-Tests'
+  describe "making an inactive POM active", vcr: { cassette_name: 'prison_api/edit_poms_activate_pom_feature' } do
+    let(:prison) { Prison.find('LEI') }
+    let(:other_prison) { create(:prison) }
 
-    click_link "Edit profile"
+    before do
+      # This is the ID for MOIC Integration-tests
+      create(:pom_detail, :inactive, prison: other_prison, nomis_staff_id: 485_758)
+      create(:pom_detail, :inactive, prison: prison, nomis_staff_id: 485_758)
+    end
 
-    expect(page).to have_css('h1', text: 'Edit profile')
+    it 'makes the pom have a status of active' do
+      visit "/prisons/LEI/poms"
+      click_link "Inactive staff (1)"
+      click_link 'Moic Integration-Tests'
+
+      click_link "Edit profile"
 
-    find('label[for=working_pattern-5]').click
-    find('label[for=status-1]').click
+      expect(page).to have_css('h1', text: 'Edit profile')
 
-    click_button('Save')
+      find('label[for=status-1]').click
+      find('label[for=part-time-conditional-1]').click
+      find('label[for=working_pattern-5]').click
 
-    expect(page).to have_content('0.5')
-    expect(page).to have_content('Active')
+      click_button('Save')
+
+      expect(prison.pom_details.find_by(nomis_staff_id: 485_758).status).to eq('active')
```

Comment:
It might be nice to check that the same POM in `other_prison` is still marked as "inactive" – to prove that we haven't just activated them globally.